src/midi/midi_fluidsynth.cpp

```diff
@@ -465,6 +482,15 @@ bool MidiHandlerFluidsynth::Open([[maybe_unused]] const char *conf)
 		mixer_channel->SetLowPassFilter(FilterState::Off);
 	}
 
+	// Double the baseline PCM prebuffer because MIDI is demanding
+	const auto render_ahead_ms = MIXER_GetPreBufferMs() * 2;
+
+	static constexpr auto max_frames_per_ms = 48;
+	frame_fifo.Resize(render_ahead_ms * max_frames_per_ms);
+
+	static constexpr auto max_midi_msg_per_ms = 255;
+	work_fifo.Resize(render_ahead_ms * max_midi_msg_per_ms);
```

Comment:
Like in the MT-32 file, I think this limit will get us into trouble.

Reply:
Done.